ouroboros-consensus/src/Ouroboros/Consensus/Ledger/SupportsMempool.hs

```diff
@@ -93,6 +95,14 @@ class ( UpdateLedger blk
   -- annotation.
   txInBlockSize :: GenTx blk -> Word32
 
+  -- | The maximum number of ExUnits worth of transactions that can be put into
```

Comment:
See my response in this thread: https://github.com/input-output-hk/ouroboros-network/pull/3224/files#r657368550